### ISSUE TITLE
data wow delay

### ISSUE THREAD

**Issue by samuele1990** [matthieua/WOW#200]
Hi!
Can you explane why in mobile devices data-wow-delay doesn't run?

**Comment by vyskoczilova** [comment]
Yep, I would like to know as well, or at least - how to disable data wow for mobile phones?

**Comment by sayhien** [comment]
Hello matthieua, I really appreciate your awesome work developing animate.css and wow.js etc. Many of us web designers/developers owe a great deal to you!

Anyway, I have no problems with delay; but offset cannot work right now.

Perhaps time for an update?

Thanks again.

**Comment by dongepulango** [comment]
Same here.
